fetch.js

```diff
@@ -236,6 +236,21 @@
       }
     }
 
+    if (support.arrayBuffer) {
+      this.arrayBuffer = function() {
+        if (this._bodyArrayBuffer) {
+          var rejected = consumed(this)
+          if (rejected) {
+            return rejected
+          }
+
+          return Promise.resolve(this._bodyArrayBuffer.buffer)
+        } else if (support.blob) {
```

Comment:
I wouldn't test for Blob support here. First of all, do any browsers implement ArrayBuffer but not Blob? Second, I don't want this function to ever return `undefined`.